package.json

```diff
@@ -27,14 +58,27 @@
     "node": "*"
   },
   "scripts": {
-    "lint": "eslint .",
-    "test": "eslint . && istanbul cover ./node_modules/mocha/bin/_mocha -- -R dot tests",
-    "browserfiles": "./bin/bundle"
+    "babel": "babel nunjucks --out-dir .",
+    "bundle": "node scripts/bundle.js",
+    "build": "npm run babel && npm run bundle",
+    "codecov": "codecov",
+    "mocha": "mocha -R spec tests",
+    "instrument": "cross-env NODE_ENV=test scripts/bundle.js",
```

Comment:
Would it be more obvious to call script `bundle:test`?

Reply:
Or maybe `test:bundle`.

Same is for `codecov` and `mocha`. Probably would be better to group them under `test:mocha` and `test:coverage`